.gitignore

```diff
@@ -5,6 +5,9 @@ demo/.env
 var/*
 tmp/*
 
+# Used by rake by
+/bin/by
```

Comment:
huh, a leading slash? is our other gitignore stuff wrong, in the sense that it's relative?

Reply:
If you only want to ignore in the root directory, the other entries should have a leading slash as well.  If you want to ignore the names in any directory, then the entries should not have a leading slash.

Reply:
I can work on a .gitignore patch after I merge this and #2110